### PR TITLE
🍰 feature/admin_panel

### PR DESCRIPTION
**Pull request by Olderestin**
## What changed?     
Add models to the django admin panel.

## Checklist before requesting a review     
- [x] I have performed a self-review of my code
- [x] Code is functional on my machine(Windows 10/Debian GNU/Linux 11 (bullseye))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +82,10 @@ ___
    ```bash
     docker exec -it django python manage.py migrate
 
+3. Create superuser:
+   ```bash
+   docker exec -it django python manage.py createsuperuser
```
Comment (memhard) [inline README.md]: this step does not apply to installing the application

File: team/admin.py [hunk 2]
```diff
@@ -1,3 +1,26 @@
 from django.contrib import admin
 
+from team.models import Member, Team
+
 # Register your models here.
+class TeamMemberInline(admin.TabularInline):
```
Comment (memhard) [inline team/admin.py:6]: Add docs

File: team/admin.py [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+    model = Team.members.through
+    extra = 1
+    fields = ['member']
+
+class MemberTeamInline(admin.TabularInline):
```
Comment (memhard) [inline team/admin.py:17]: Add docs

File: team/admin.py [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+    model = Member.teams.through
+    extra = 1
+    fields = ['team']
+
+@admin.register(Team)
+class TeamAdmin(admin.ModelAdmin):
```
Comment (memhard) [inline team/admin.py:29]: Add docs

File: team/admin.py [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+    list_display = ['title', 'description']
+    inlines = [TeamMemberInline]
+    exclude = ['members']
+
+@admin.register(Member)
+class MemberAdmin(admin.ModelAdmin):
```
Comment (memhard) [inline team/admin.py:41]: Add docs

**Review by memhard — COMMENTED** [review]
fix docs and can approve!

**Review by Olderestin — COMMENTED** [review]
Task done.

**Review by memhard — APPROVED** [review]
LGTM 👍

**Pull request merged**